File: Sesion-07/Reto-01/AppObjC/AppObjC/ViewController.m
//
//  ViewController.m
//  AppObjC
//
//  Copyright © 2019 Bedu. All rights reserved.
//

#import "ViewController.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
  [super viewDidLoad];
  // Do any additional setup after loading the view.
}

- (IBAction)calculate:(id)sender {
  int value1 = _textField1.text.intValue;
  int value2 = _textField2.text.intValue;
  int result = value1 + value2;
  _label.text = [[NSString alloc] initWithFormat: @"%d", result];
  
  if (value1 == value2) {
    _label.textColor = [UIColor greenColor];
  } else {
    _label.textColor = [UIColor redColor];
  }
}
@end

File: Sesion-07/Reto-02/CustomButton/CustomButton/ViewController.swift
//
//  ViewController.swift
//  CustomButton
//
//  Copyright © 2019 Bedu. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

  @IBOutlet weak var button: UIButton!
  
  override func viewDidLoad() {
    super.viewDidLoad()
    CustomButton.setup(button)
    // Do any additional setup after loading the view.
  }


}

